opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -44,17 +44,14 @@ pub struct LoggerProvider {
     inner: Arc<LoggerProviderInner>,
 }
 
-/// Default logger name if empty string is provided.
-const DEFAULT_COMPONENT_NAME: &str = "rust.opentelemetry.io/sdk/logger";
-
 impl opentelemetry::logs::LoggerProvider for LoggerProvider {
     type Logger = Logger;
 
     fn logger(&self, name: impl Into<Cow<'static, str>>) -> Self::Logger {
-        let mut name = name.into();
+        let name = name.into();
 
         if name.is_empty() {
-            name = Cow::Borrowed(DEFAULT_COMPONENT_NAME)
+            otel_warn!(name: "LoggerProvider.Logger.EmptyName",  message = "Logger name is empty; consider providing a meaningful name");
```

Comment:
```suggestion
            otel_info!(name: "LoggerNameEmpty",  message = "Logger name is empty; consider providing a meaningful name. Logger will function normally and the provided name will be used as-is.");
```

Reply:
shouldn't this be otel_warn, as the end-user has provided *invalid* name?

Reply:
Okay with warn as well. Since the telemetry is still going to flow, I though maybe Info is also okay. (if we were going to drop logs from this logger, definitely Warn)

Reply:
Also - its not end-users who provided invalid information - its the appender authors, right?

Reply:
yes for tracing-appender. Maybe some custom appender allows user to configure it.

Reply:
I think this should be Info only, as telemetry is flowing without issues, and is merely a suggestion to user to provide better names.